### ISSUE TITLE
`pip check` reports version number as `0.0.0`

### ISSUE THREAD

**Issue by jan-janssen** [conda-forge/autoray-feedstock#17]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

`blueqat` depends on `autoray` and currently the `pip check` fails with the error message: 
```
blueqat 2.0.4 has requirement autoray>=0.2.5, but you have autoray 0.0.0.
```
So it seems the version from `pypi` is not transferred correctly. 

### Installed packages

```shell
-
```

### Environment info

```shell
-
```

**Comment by jan-janssen** [comment]
As reference https://github.com/conda-forge/blueqat-feedstock/pull/13

**Comment by jcmgray** [comment]
I *suspect* the resolution is to have `setuptools_scm` specified in the host requirements, I'll try and fix tomorrow.

**Comment by jcmgray** [comment]
Hopefully this is fixed by #18, let me know!

**Comment by jan-janssen** [comment]
> Hopefully this is fixed by #18, let me know!

Perfect - thanks a lot
